lib/sisimai/bite/email/apachejames.rb

```diff
@@ -0,0 +1,169 @@
+module Sisimai::Bite::Email
+  module ApacheJames
+    # Sisimai::Bite::::Email::ApacheJames parses a bounce email which created by
+    # ApacheJames. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/ApacheJames.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->